client/v2/src/test/java/org/opennms/velocloud/client/v2/VelocloudApiClientV2Test.java

```diff
@@ -38,20 +38,34 @@
 import static org.junit.Assert.*;
```

Comment:
I probably missed this as well. It would be better if we just can just import what we are using.